client/testserver/src/main/scala/sttp/tapir/client/tests/HttpServer.scala

```diff
@@ -133,8 +133,27 @@ class HttpServer(port: Port) {
           val e = q.enqueue
           WebSocketBuilder[IO].build(d, e)
         }
+
+    case r @ GET -> Root / "api" / "organization" =>
+      fromAcceptHeader(r) {
+        case "application/json" => organizationJson
+        case "application/xml"  => organizationXml
+      }
+
+    case r @ GET -> Root / "api" / "entity" =>
+      fromAcceptHeader(r) {
+        case "application/json" => personJson
+        case "application/xml"  => organizationXml
+      }
   }
 
+  private def fromAcceptHeader(r: Request[IO])(f: String => IO[Response[IO]]): IO[Response[IO]] =
+    r.headers.get(Accept).map(h => f(h.value)).getOrElse(NotAcceptable())
+
+  private val organizationXml = Ok("<name>sml</name>", `Content-Type`(MediaType.application.xml))
```

Comment:
maybe here we could return `<name>sml-xml</name>` so that later in the test we coudl verify that indeed, we got the xml result that we wanted, not another one.